modules/manage/pages/manage-settings/general-settings.adoc

```diff
@@ -97,14 +97,18 @@ image::manage-settings/node-availability.png["The Node Availability panel",548,a
 
 The following checkboxes are provided:
 
-* *Auto-failover after _x_ seconds for up to _y_ node*: After the timeout period set here as _x_ seconds has elapsed, an unresponsive or malfunctioning node is failed over, provided that the limit on actionable events set here as _y_ has not yet been reached.
+* *Auto-failover after _x_ seconds for up to _y_ node*: After the timeout period set here as _x_ seconds has elapsed, an unresponsive or malfunctioning node is failed over, provided that the limit on actionable events set here as _y_ (with the default value of 1) has not yet been reached.
 Data replicas are promoted to active on other nodes, as appropriate.
 This feature can only be used when three or more nodes are present in the cluster.
 The number of seconds to elapse is configurable: the default is 120; the minimum permitted is 5; the maximum 3600.
 This option is selected by default.
+
 * *Auto-failover for sustained data disk read/write failures after _z_ seconds*: After the timeout period set here as _z_ seconds has elapsed, a node is failed over if it has experienced sustained data disk read/write failures.
-The timeout period is configurable: the default length is 120 seconds; the minimum permitted is 5; the maximum 3600.
+The timeout period is configurable: the default length is 240 seconds; the minimum permitted is 5; the maximum 3600.
 This checkbox can only be checked if *Auto-failover after _x_ seconds for up to _y_ node* has also been checked.
+This option is selected by default.
+
+* *Preserve durable writes*: If this checkbox is checked (which is the default), and the node to be automatically failed over is a Data Service node that hosts data for a bucket whose Minimum Durability Level is greater than *none*; auto-failover occurs only if xref:learn:data/durability.adoc#majority[Majority] can be maintained across the nodes on which the bucket's data resides.
```

Comment:
The default is unchecked I believe. 
Also, the check is performed on all buckets regardless of Minimum Durability Level. 
CC @BenHuddleston  to correct me if I'm mistaken.

Reply:
Neither the REST or the CLI docs specify exactly the default - it is rather implied as 'none'. (So I should clarify whether the default is indeed 'none'.) At any rate, both say 'none' means asynchronous writes, and I guess I assumed that that meant 'majority' was not an issue. 

(NB The UI for bucket creation provides a pull-down menu whose default is 'none'.)

Reply:
Default should be that we don't perform this check to preserve durable writes to maintain the current behaviour (i.e. unchecked). I didn't do the UI work but testing this locally had this default to unchecked.

As Abhi said, Minimum Durability Level is irrelevant here.